src/common/components/Day.vue

```diff
@@ -0,0 +1,173 @@
+<template lang="pug">
+  .wrap
+    span.skeleton-loader(
+      v-if="isLoading" 
+      v-for="_ in countDay"
+    )
+
+    div(
+      v-if="!isLoading"
+      v-for='day in countDay' 
+      :class='{ day: (dates[day - 1] !== undefined && dates[day - 1].thisMonth), none: !(dates[day - 1] !== undefined && dates[day - 1].thisMonth), selected: (dates[day - 1] !== undefined && dates[day - 1].isSelected) }' 
+      @click='dayClick(dates[day - 1] !== undefined && dates[day - 1].thisMonth && dates[day - 1].isPast, dates[day - 1].date, dates[day - 1].index)'
+    )
+      div(
+        v-if="dates[day - 1] !== undefined && dates[day - 1].thisMonth"
+      )
+        svg.today(
```

Comment:
could you please move this to `asset` for now ? also don't forget to register the new icons on debio-icons, and adjust later once the repo has been published ?